src/client/kotlin/tech/thatgravyboat/skyblockapi/utils/extentions/StringExtensions.kt

```diff
@@ -192,3 +192,9 @@ fun String.trimIgnoreColor(): String {
 }
 
 fun UUID.toDashlessString(): String = toString().replace("-", "")
+
+fun String.removeTrailingChar(target: Char): String {
+    var i = lastIndex
+    while (i >= 0 && this[i] == target) i--
+    return if (i < lastIndex) substring(0, i + 1) else this
+}
```

Comment:
```suggestion
fun String.removeTrailingChar(target: Char): String = dropLastWhile {  it == target }
```

i think this should be the same (mona agreed)